llvm/lib/Target/AIE/AIEPostPipeliner.cpp

```diff
@@ -641,10 +653,10 @@ class ConfigStrategy : public PostPipelinerStrategy {
   ConfigStrategy(ScheduleDAGInstrs &DAG, std::vector<NodeInfo> &Info,
                  int Length, enum Modulation Mode, bool FromTop, uint64_t Param)
       : PostPipelinerStrategy(DAG, Info, Length), Mode(Mode), FromTop(FromTop),
-        Param(Param), RandomState(spice(Param)) {
-    if (Mode == Modulation::Split) {
-      Count = Length * Param / 1000;
-    }
+        Param(Param) {
+    Priority[0] = (Param >> 0) & 0x3;
+    Priority[1] = (Param >> 2) & 0x3;
+    Priority[2] = (Param >> 4) & 0x3;
```

Comment:
Not really elegant. Inspired by an opaque chromosome value to be used in a genetic algorithm.